.azure/gpu-benchmark.yml

```diff
@@ -46,19 +46,43 @@ jobs:
 
     - bash: |
         echo "##vso[task.setvariable variable=CUDA_VISIBLE_DEVICES]$(DEVICES)"
-        cuda_ver=$(python -c "import torch ; print(''.join(map(str, torch.version.cuda.split('.')[:2])))")
-        echo "##vso[task.setvariable variable=CUDA_VERSION_MM]$cuda_ver"
-        echo "##vso[task.setvariable variable=TORCH_URL]https://download.pytorch.org/whl/cu${cuda_ver}/torch_stable.html"
+        echo "##vso[task.setvariable variable=TORCH_URL]https://download.pytorch.org/whl/cu116/torch_stable.html"
```

Comment:
The cuda_ver variable here did not have a value. It caused torchvision to be installed with +cu117 but we needed cu116 to match the torch version. Hardcoding it here since I didn't know what else to do

Reply:
You can copy it from the other workflows. Removing it was probably an accident

Reply:
that worked!